resources/js/components/Product/AddToCart.vue

```diff
@@ -402,76 +405,49 @@ export default {
         },
 
         disabledOptions: function () {
```

Comment:
Are we **certain** this change does not alter behavior in any way.
This code has gone through quote some iteration to make sure it exactly matches Magento behavior. As any deviation can cause errors when adding to cart

Reply:
Right now it's a very simple approach which probably does alter behavior a bit. I tried to keep the UX simple but it's actually not very straightforward for this if the user decides to not go in order of the options. I'm not sure what the best approach is.

However, as for getting errors when adding to cart, as long as you don't end up being able to select a non-existent set of options I don't see how errors are possible.

Reply:
I've altered this function to match the expected functionality, where you can only ever click options that don't result in an invalid state. Because you can't un-click options this does mean you can get stuck in a sort of "maze" of non-disabled options. But that's the intended functionality 😅